src/containers/Homepage/index.js

```diff
@@ -27,7 +27,7 @@ import {
   StyledHeroValueProposition
 } from './components';
 
-const Homepage = props => {
+const Homepage = (props) => {
```

Comment:
eslint rule mentioned above!

Reply:
always presenting parens make refactoring easier: when you need to add another arg you don't need to add parens (cause they are present already).